meta-oe/recipes-extended/rsyslog/rsyslog_8.2102.0.bb

```diff
@@ -151,7 +151,7 @@ do_install_append() {
     if ${@bb.utils.contains('DISTRO_FEATURES','systemd','true','false',d)}; then
         install -d ${D}${systemd_system_unitdir}
         install -m 644 ${WORKDIR}/rsyslog.service ${D}${systemd_system_unitdir}
-        sed -i -e "s,@sbindir\@,$(sbindir),g" ${D}${systemd_system_unitdir}/rsyslog.service
+        sed -i -e 's,\@sbindir\@,'"$sbindir"',g' ${D}${systemd_system_unitdir}/rsyslog.service
```

Comment:
this seems to work ok here, I am using `bash` for `/bin/sh`, whats your default `/bin/sh` can you try 
```
sed -i -e 's,@sbindir@,${sbindir},g' ${D}${systemd_system_unitdir}/rsyslog.service
```
and see if that works ?

and also check the sed operation on line 141, I think that might need some care too.

Reply:
Yeah, good point, it's really just the () vs. {} causing the issue. I verified 141 works fine because it has the correct {}